.github/workflows/neon_extra_builds.yml

```diff
@@ -38,7 +38,7 @@ jobs:
           fetch-depth: 1
 
       - name: Install macOS postgres dependencies
-        run: brew install flex bison openssl protobuf
+        run: brew install flex bison openssl protobuf curl
```

Comment:
I think this is not used right now. `curl` from Homebrew is `keg only` which means it's not symlinked anywhere by default and with a high probability our build system doesn't use it.
MacOS comes with bundled curl (7.87.0 on macOS 13.3.1), I guess this one is got used.